compose/galaxy-htcondor-base/Dockerfile

```diff
@@ -18,3 +18,8 @@ RUN wget https://dl.influxdata.com/telegraf/releases/telegraf-1.5.0_linux_amd64.
     rm telegraf-1.5.0_linux_amd64.tar.gz
 
 ADD telegraf.conf /etc/telegraf/telegraf.conf
+
+
+RUN touch /var/log/condor/StartLog /var/log/condor/StarterLog /var/log/condor/CollectorLog /var/log/condor/NegotiatorLog && \
```

Comment:
This can be removed now I think.

Reply:
Just tested it: I think it's still needed as this image is not based on galaxy-base. I get errors at startup without this line